libs/scripts/scripts.js

```diff
@@ -19,6 +19,8 @@ import {
 // Production Domain
 const prodDomains = ['milo.adobe.com'];
 
+const stageDomainsMap = { 'www.adobe.com': 'www.stage.adobe.com' };
```

Comment:
IMO we should add all known envs here, a lot of the script-configs are simply copy pasted from milo from what I've seen.

Reply:
My reasoning was to keep it as short as possible because it won't be used on production. Anyway, if consumers copy paste this for their script it would make sense to have a more complete list. Will add more envs 👍